v2/setup/marathon.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+source /etc/environment
+
+HOMEDIR=$(eval echo "~`whoami`")
+
+sudo docker run --rm \
+    -v ${HOMEDIR}:/data/ behance/docker-aws-s3-downloader \
+     us-east-1 $CONTROL_TIER_S3SECURE_BUCKET .marathon
+
+SYSDIG_KEY=$(sudo cat ${HOMEDIR}/.marathon)
+
+etcdctl set /marathon_user $marathon_user
+etcdctl set /marathon_password $marathon_password
```

Comment:
how are `$marathon_user` and `$marathon_password` being set?
I don't know bash too well.